examples/ex6p.cpp

```diff
@@ -2,20 +2,21 @@
 //
 // Compile with: make ex6p
 //
-// Sample runs:  mpirun -np 4 ex6p -m ../data/square-disc.mesh -o 1
+// Sample runs:  mpirun -np 4 ex6p -m ../data/square-disc.mesh -o 1 -cs
+//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -o 2 -cs
 //               mpirun -np 4 ex6p -m ../data/square-disc.mesh -o 2
-//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -o 2 -ns
 //               mpirun -np 4 ex6p -m ../data/square-disc-nurbs.mesh -o 2
 //               mpirun -np 4 ex6p -m ../data/star.mesh -o 3
+//               mpirun -np 4 ex6p -m ../data/escher.mesh -o 2 -cs
 //               mpirun -np 4 ex6p -m ../data/escher.mesh -o 2
-//               mpirun -np 4 ex6p -m ../data/escher.mesh -o 2 -ns
 //               mpirun -np 4 ex6p -m ../data/fichera.mesh -o 2
 //               mpirun -np 4 ex6p -m ../data/disc-nurbs.mesh -o 2
 //               mpirun -np 4 ex6p -m ../data/ball-nurbs.mesh
 //               mpirun -np 4 ex6p -m ../data/pipe-nurbs.mesh
 //               mpirun -np 4 ex6p -m ../data/star-surf.mesh -o 2
-//               mpirun -np 4 ex6p -m ../data/square-disc-surf.mesh -o 2
+//               mpirun -np 4 ex6p -m ../data/square-disc-surf.mesh -o 2 -cs
 //               mpirun -np 4 ex6p -m ../data/amr-quad.mesh
+//               mpirun -np 4 ex6p -m ../data/inline-segment.mesh -o 1 -md 200
 //               mpirun -np 4 ex6p --restart
```

Comment:
I think we want to keep the sample run `mpirun -np 4 ex6p --restart` immediately after `mpirun -np 4 ex6p -m ../data/amr-quad.mesh`. Right now it will restart with the mesh from `mpirun -np 4 ex6p -m ../data/inline-segment.mesh -o 1 -md 200` and I don't think we want that.

Reply:
OK, I didn't think of that...